scripts/rebuild-ui.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd ../jaeger-ui
+
+if git describe --exact-match --tags $(git rev-parse HEAD) >/dev/null 2>&1; then
```

Comment:
don't think you need `$(git rev-parse HEAD)`, by default it gives you current commit